BaseClass.java

```diff
@@ -0,0 +1,20 @@
+package com.automationpractice;
+
+import org.junit.BeforeClass;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public abstract class BaseClass {
+
+    protected static WebDriver chrome;
+
+    @BeforeClass
+    public static void setup(){
+        System.setProperty("webdriver.chrome.driver", "C:\\Program Files\\TOOLS\\chromedriver_win32\\chromedriver.exe");
```

Comment:
avoid hardcoded value to the driver file, in examples, you can find how to read from the properties file or from the environment.